helpers.js

```diff
@@ -33,6 +34,20 @@ const specNameMap = {
     Windwalker: 'WW'
 };
 
+const sayQuotes = [
+    'Many times have I fallen at the hands of an orc and many times have I risen again.',
+    'For every orc that struck me, I will cleave a thousand of their skulls! This ends here!',
+    'For the Horde, my friend.',
+    'I may be bruised and beaten, but the hatred boils inside me.'
+];
+
+const shoutQuotes = [
+    'THIS ENDS HERE!',
+    `<:gamon_shout:${GAMON_SHOUT_ID}> I, GAMON, WILL SAVE US!`,
+    ':axe: TASTE MY AXE!',
+    'I AM A STORM OF PAIN!',
```

Comment:
Hmm can we get the storm bolt icon from warrior as an emoji for this?

Reply:
Done and done!